.github/actions/publish-nuget/action.yml

```diff
@@ -34,10 +34,9 @@ runs:
     - name: Setup Scripts
       shell: pwsh
       run: |
-        "${{ github.action_path }}" >> $Env:GITHUB_PATH
```

Comment:
As I've written, this step was good. It's good to have a separate step just to set up scripts, including for the action.

Reply:
Fixed this.